### ISSUE TITLE
Does table in radish support table header?

### ISSUE THREAD

**Issue by tangorboyz** [radish-bdd/radish#185]
I mean we could the define a table in feature file like this:
```
| username | email                   | password |
| john     | john@example.com        | anything |
```
And we could treat each row like a python dictionary. We could access each cell like `table[0]['username'] `I try this before but:
`TypeError: list indices must be integers or slices, not str`
And then i just use` table[0][0]`, which is not so readable.

**Comment by timofurrer** [comment]
Sounds like a reasonable change. I've just checked and without the header it's not really valid gherkin.
So we should do this.

I'm happy to review a PR if you'd like to make the change yourself!

**Comment by timofurrer** [comment]
I've implemented it in #187

**Comment by timofurrer** [comment]
It's released with **v0.8.0** :tada:

**Comment by tangorboyz** [comment]
Ok, cool
